spring-cloud-starter-stream-sink-jdbc/src/main/java/org/springframework/cloud/stream/app/jdbc/sink/JdbcSinkConfiguration.java

```diff
@@ -83,7 +103,29 @@
 	private EvaluationContext evaluationContext;
 
 	@Bean
+	public MessageChannel toSink() {
```

Comment:
We don't need this bean.
The `inputChannel = "toSink"` will take care for channel creation for us.
Also you can use a `AggregatorFactoryBean.setOutputChannelName()`  instead.

Reply:
I tried removing this bean and calling setOutputChannelName("toSink"), but keep getting an `UnsatisfiedDependencyException` with message " Unsatisfied dependency expressed through method 'aggregatorFactoryBean' parameter 0; nested exception is org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type 'org.springframework.messaging.MessageChannel' available: expected single matching bean but found 3: input,nullChannel,errorChannel" on each test.

Could you please explain how to refactor this? Unfortunately, I'm new to Spring in general.